tikv-client/src/main/java/com/pingcap/tikv/meta/TiDAGRequest.java

```diff
@@ -218,26 +242,21 @@ private DAGRequest buildTableScan() {
     // DO NOT EDIT EXPRESSION CONSTRUCTION ORDER
     // Or make sure the construction order is below:
     // TableScan/IndexScan > Selection > Aggregation > TopN/Limit
-    TiExpr filterExpr = mergeCNFExpressions(
-        getFilter().stream().peek(this::setColumnOffsets).collect(Collectors.toList())
-    );
-
```

Comment:
There's one PR's logic omitted here, please refer to https://github.com/pingcap/tikv-client-lib-java/pull/203